### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by trca831**
Need additional assistance with prompter.js

### CODE REVIEW

File: 01-node-tutorial/answers/10-fs-sync.js
Comment (jamisonvalentine) [inline 01-node-tutorial/answers/10-fs-sync.js:1]: The order is wrong here: you want to write to the file then read it... but I think you get the concept.

**Review by jamisonvalentine — APPROVED** [review]

**Review by leo-mutuku — CHANGES_REQUESTED** [review]
Hi Tracy, I have reviewed the assignment, well done and clearly understood the concepts. However I recomend a 1:1 review especially on node modules. I will be reviewing the lesson on my 1:1 section friday and satarday.

File: 01-node-tutorial/answers/prompter.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/prompter.js:1]: Approved

**Review by leo-mutuku — APPROVED** [review]
Week1 assessment completed.

**Review by GinaCastromonte — COMMENTED** [review]
Great Job Tracy! Looks like you adjusted your changes.

**Pull request merged**